builtin/providers/aws/resource_aws_vpc_peering_connection_accept.go

```diff
@@ -0,0 +1,64 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsVpcPeeringConnectionAccept() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsVPCPeeringAcceptCreate,
+		Read:   resourceAwsVPCPeeringAcceptRead,
+		Delete: resourceAwsVPCPeeringAcceptDelete,
+
+		Schema: map[string]*schema.Schema{
+			"peering_connection_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"accept_status": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceAwsVPCPeeringAcceptCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).ec2conn
+
+	if cur, ok := d.Get("accept_status").(string); ok && cur == ec2.VpcPeeringConnectionStateReasonCodeActive {
+		// already accepted
+		return nil
+	}
+
+	status, err := resourceVPCPeeringConnectionAccept(conn, d.Id())
+	if err != nil {
+		return err
+	}
+	d.Set("accept_status", status)
+
+	// TODO: should we poll until this resolves? VpcPeeringConnectionStateReasonCodePendingAcceptance
```

Comment:
Yes, it didn't work for me until after I added
`	stateConf := &resource.StateChangeConf{
		Pending: []string{"pending-acceptance"},
		Target:  []string{"active"},
		Refresh: resourceAwsVPCPeeringConnectionStateRefreshFunc(conn, d.Id()),
		Timeout: 1 * time.Minute,
	}

	if _, err := stateConf.WaitForState(); err != nil {
		return errwrap.Wrapf(fmt.Sprintf(
			"Error waiting for VPC Peering Connection Accept (%s) to become available: {{err}}",
			d.Id()), err)
	}`